### ISSUE TITLE
Tune-after + MTS-ESP

### ISSUE THREAD

**Issue by Andreya-Autumn** [surge-synthesizer/surge#7236]
...isn't possible to do in a fully robust way, since modulations can bring the pitch params outside the range MTS-ESP provides and without knowing the scale size, we can't know for sure what the next frequencies should be. (The ratio of MIDI note 128 to 129 is somewhere in the table, but where?).

But today in discussion about this, Paul said:
"The other thing I could do right now is
Assume the ratio between 60 and 72 is representative of a 12 note repetition interval
Extend outside 128 using that
Write in the manual that mts tune after will have approximations below 8hz and above 16000
Sweep the full table every 32 blocks
Add a menu item
And call it a day"

Which for any scale with ≈12 notes or fewer per approximately octave-sized span (on a reasonable reference freq and note) will probably work perfectly fine honestly.

**Comment by baconpaul** [comment]
I'm gonna bounce this out of 1.3 so we can get a release stabilized.

**Comment by mkruselj** [comment]
Looking at #8138 here seems to be this issue is heavily tied to it.

**Comment by Andreya-Autumn** [comment]
Indeed. I'm gonna just close this one now. The other one is more relevant I think.
